### ISSUE TITLE
linter: Oxc vscode extension need save .oxlintrc.json manually to make it work in nested project

### ISSUE THREAD

**Issue by ChangeHow** [oxc-project/oxc#11217]
### What version of Oxlint are you using?

0.16.8

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  // determine that maybe effect by node_modules
  // "extends": ["./node_modules/@creation-tools/eslint-config/dist/configs/oxlint-base.json"],
  "extends": ["./oxlint-base.json"],
}
```

### What happened?

I have a project like:
```yaml
- root
  - .oxlintrc.json # to make oxlint workaround
  - repoA
    - .oxlintrc.json # <--- using node_modules/lint/config/oxlint-base.json
```

**Issue Observed:**
Oxc doesn't report any issues after:

- Reloading VSCode
- Restarting Windsurf
- Running trace commands\
(Note: This behavior also occurred in earlier versions 0.15.x...)

**Workaround action:**
But everything work well after I saved `repoA/.oxlintrc.json` manually.

Here is the screen recording:
https://github.com/user-attachments/assets/431df776-cde3-4dd0-89d1-01578f706c36

**Comment by ChangeHow** [comment]
any update for this issue?

**Comment by Sysix** [comment]
With 0.16.8 I got the same problem.
Try version 0.16.11, it works for me

![Image](https://github.com/user-attachments/assets/0d466b54-07cf-4982-b0ec-27d2c3d9488c)

**Comment by ChangeHow** [comment]
> With 0.16.8 I got the same problem.
> Try version 0.16.11, it works for me
> 
> ![Image](https://github.com/user-attachments/assets/0d466b54-07cf-4982-b0ec-27d2c3d9488c)
> 
> I think this was the bug and got fixed in 0.16.9:
> https://github.com/oxc-project/oxc/pull/10698

Let me try this version later.

**Comment by ChangeHow** [comment]
It works well in my project after upgraded to the latest version.

Thank you for your job @Sysix 👍
